lib/vtex/client.rb

```diff
@@ -21,7 +21,8 @@ def connection
 
     def self.resources
       {
-        products: ProductResource
+        products: ProductResource,
+        categories: CategoryAndBrandResource
```

Comment:
not sure if this should be called `categories_and_brands` or just categories ... what do you think????